concepts/symbol/about.md

```diff
@@ -5,32 +5,21 @@ Symbols are created through a symbol literal, which is by prefixing a name with
 They also allow for being written with quotes, e.g. `:"foo"`, which allows, for example, spaces in the name.
 
 ```crystal
-:foo # => :foo
-:"foo boo" # => :"foo boo"
+:foo           # => :foo
+:"foo boo"     # => :"foo boo"
+:foo == :"foo" # => true
 ```
 
 Symbols are used in many places in the language, including as keys in namedtuples and to represent method and variable names.
 
 ## Symbols in Crystal
 
 Symbols in Crystal are quite different from Ruby.
-In Crystal a symbol is a type of constant and is thereby is assigned at compile time.
+In Crystal a symbol is a type of constant and thereby is assigned at compile time.
 This means that symbols can't be created dynamically, which is possible in Ruby.
 
 Symbols in Crystal are represented as `Int32`s which makes them very efficient.
 
-## Identifier
-
-What makes symbols different from strings is that they are identifiers and do not represent data or text.
-This means that two symbols with the same name are always the same object.
-
-```ruby
-"foo".object_id # => 60
-"foo".object_id # => 80
-:foo.object_id # => 1086748
-:foo.object_id # => 1086748
-```
-
```

Comment:
This is valid in Ruby, but I couldn't find a way to show this in Crystal, so I removed this part.